CHANGELOG.md

```diff
@@ -13,9 +13,10 @@ c2d4164](https://github.com/nf-core/coproid/pull/20/commits/c2d4164bf068ed4fc92d
 - Update documentation [bedfdde](https://github.com/nf-core/coproid/commit/bedfddec8500adac8e0cb9cc8e0df2dc6a784f15)
 - Update Nextflow minimum version to 19.04.0 [44999fd](https://github.com/nf-core/coproid/commit/44999fd4d38b21d53f970621dbf3587c044da8d1)
 - Update travis for more recent nextflow requirements [1e3e454](https://github.com/nf-core/coproid/commit/1e3e454e72f1bc8eb43aaa1e5165981cb77a56dc)
-- Adapt coproID to nf-core tools 1.8 release [#21](https://github.com/nf-core/coproid/pull/21) 
+- Adapt coproID to nf-core tools 1.8 release [#21](https://github.com/nf-core/coproid/pull/21)
 - Add social preview image [#22](https://github.com/nf-core/coproid/pull/22)
 - Fix Kraken2 segmentation error [#26](https://github.com/nf-core/coproid/pull/26)
+- Update to nf-core tools 1.9 release, and doc for new version of sphinx [#27](https://github.com/nf-core/coproid/pull/27)
```

Comment:
Any reason why you'reusing readthedocs and not standard nf-core document pages? Or is this just a leftover from when you started the pipeline? Regardless, on the docs this should be updated: https://coproid.readthedocs.io/en/latest/introduction.html#how-to-cite-coproid

Reply:
I'm using both. The https://coproid.readthedocs.io is based on the docs in the master branch. readthedocs will be updated when 1.1 is merged in master.